DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/FlagRush_RpcEvents.Script.txt

```diff
@@ -0,0 +1,280 @@
+// #RequireContext CSmMode
+
+#Include "Libs/Nadeo/ModeLibs/Legacy/XmlRpc2.Script.txt"							 	as XmlRpc
+#Include "Libs/Zrx/ModeLibs/Common/TeamPointsProgression.Script.txt" 		as TeamPointsProgression
+#Include "Libs/Nadeo/TMxSM/Race/Scores.Script.txt"											as Scores
+#Include "Libs/Zrx/ModeLibs/FlagRush/FlagRush_Common.Script.txt"				as FlagRush_Common
+#Include "Libs/Zrx/ModeLibs/Common/PlayerTelemetry.Script.txt" 					as Telemetry
+
+// XMLRPC events
+#Const		C_XMLRPC_FlagSteal															"FlagRush.Flag.Steal"
+#Const		C_XMLRPC_FlagDrop																"FlagRush.Flag.Drop"
+#Const		C_XMLRPC_FlagReset															"FlagRush.Flag.Reset"
+#Const		C_XMLRPC_FlagScored															"FlagRush.Flag.Scored"
+#Const 		C_XMLRPC_FlagPickup															"FlagRush.Flag.Pickup"
+#Const		C_XMLRPC_PlayerDeath														"FlagRush.PlayerDeath"
+#Const		C_XMLRPC_Overtime																"FlagRush.Overtime"
+#Const		C_XMLRPC_MatchStart															"FlagRush.Flow.MatchStart"
+#Const		C_XMLRPC_MatchEnd																"FlagRush.Flow.MatchEnd"
+#Const		C_XMLRPC_MapStart																"FlagRush.Flow.MapStart"
+#Const		C_XMLRPC_MapEnd																	"FlagRush.Flow.MapEnd"
+#Const		C_XMLRPC_RoundStart															"FlagRush.Flow.RoundStart"
+#Const		C_XMLRPC_RoundEnd																"FlagRush.Flow.RoundEnd"
+#Const 		C_XMLRPC_WarmUpStart 														"Trackmania.WarmUp.Start"
+#Const 		C_XMLRPC_WarmUpEnd 															"Trackmania.WarmUp.End"
+
+
+#Struct K_Team {
+	Integer Num;
+	Text Name;
+}
```

Comment:
Unnecessary Struct. `Num` already corresponds to Index of Teams-array or `Player.CurrentClan` and `Name` is `Teams[Index].Name`.